src/components/Navigation.css

```diff
@@ -1,5 +1,29 @@
 .navigation {
   display: flex;
   justify-content: space-around;
-  flex-grow: 1;
+  align-items: center;
+  background: rgba(255, 255, 255, 0.05);
+  box-shadow: 0 25px 45px rgba(0, 0, 0, 0.2);
+  border-radius: 5px;
+  backdrop-filter: blur(3px);
+}
+
+a {
+  color: #ffff;
+  margin: 20px;
+}
+
+.navigation__plus {
+  height: 60px;
+  width: 60px;
+}
+
+.navigation__suitcase {
+  height: 45px;
+  width: 45px;
+}
+
+.navigation__user {
+  height: 45px;
+  width: 45px;
```

Comment:
you can move this up to line 21/ 22, because you are using the same values for both, so you can say 
```
.navigation__suitcase, 
.navigation__user {
height.....
}
```

Reply:
@FriederikeM very good point to minimalize the code :)